src/lambda.rs

```diff
@@ -0,0 +1,94 @@
+mod commp;
+
+use std::error::Error;
+use std::str::FromStr;
+
+#[macro_use]
+extern crate lambda_runtime as lambda;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate log;
```

Comment:
The same as above also for the logging. Within the module you use logging, you just import the macros directly:

```rust
use log::{debug, info}
```